internal/alloycli/cmd_run.go

```diff
@@ -164,6 +165,7 @@ type alloyRun struct {
 	configFormat                 string
 	configBypassConversionErrors bool
 	configExtraArgs              string
+	community                    bool
```

Comment:
```suggestion
	enableCommunityComps                    bool
```
I don't think the current name is clear enough and it will cause more cognitive overhead when reading the code.